src/did/src/transaction.rs

```diff
@@ -184,7 +184,7 @@ impl From<EthersSignature> for Signature {
 impl Signature {
     /// Upper limit for signature S field.
     /// See comment to `Signature::check_malleability()` for more details.
-    pub const S_UPPER_LIMIT_HEX_STR: &str =
+    pub const S_UPPER_LIMIT_HEX_STR: &'static str =
```

Comment:
Isn't `static` here deduced automatically?

Reply:
Yes, but clippy says to fix it.